### ISSUE TITLE
IE11 replaceChild error

### ISSUE THREAD

**Issue by danyj** [maxwellito/vivus#205]
v0.4.4 

https://github.com/maxwellito/vivus/blob/master/dist/vivus.js#L741  is causing  

 Unable to get property 'replaceChild' of undefined or null reference

http://prntscr.com/kqvacq

**Comment by maxwellito** [comment]
Please provide an example. JSfiddle or any other platform. Thanks.

**Comment by maxwellito** [comment]
Thanks for providing a URL to test. However, I'm afraid I couldn't reproduce it as intended. Over the first visits, IE triggered no errors and the animation was smooth. However after playing many times with your scrolling system I managed to get the same error. I think it's related to something else, too much magic happening. My guess is your scroll panel system looks like it create and delete content along the scroll. And at some point it tries to trigger the animation on a deleted SVG. The error appear only on IE because by default the `forceRender` is on this browser. Forcing to recreate each path element on something that no longer exists in the DOM.

I retested the interactions from the manual tests on IE11 and didn't noticed any problem.

**Comment by danyj** [comment]
hm , is not removing anything but hiding so by your explanation and the triggered error seems like it cant find the element due to its `visibility: hidden` class being applied.

**Comment by danyj** [comment]
This is the class and the rules that element gets when out of the view 

```
.thz-animate {
    opacity: 0;
    animation-duration: 0.7s;
    -webkit-animation-duration: 0.7s;
    -moz-animation-duration: 0.7s;
    -o-animation-duration: 0.7s;
    animation-timing-function: ease-in-out;
    -webkit-animation-timing-function: ease-in-out;
    -moz-animation-timing-function: ease-in-out;
    -o-animation-timing-function: ease-in-out;
    -webkit-perspective: 1000;
    -webkit-backface-visibility: hidden;
    backface-visibility: hidden;
    will-change: transform,opacity;
}
```

**Comment by danyj** [comment]
Here , this is all it happens to the element, class added/removed http://take.ms/kCJpn

**Comment by maxwellito** [comment]
I'm afraid I can't help you more. The code of your page is minified and it's pretty hard to debug. This is also why I don't usually accept personal websites, but only sandboxed pages (from jsfiddle or custom gh-pages). Here is the [example](http://maxwellito.github.io/drafts/vivus-198/bug.html) I provided to the Chromium team when I reported a bug.

So far the manuals tests are working on IE11, and I also managed to see the animation working on your page. So I can't consider it a bug.
